Sources/PointFree/Episode/Show.swift

```diff
@@ -353,6 +368,12 @@ private let transcriptBlockView = View<Episode.TranscriptBlock> { block -> Node
       )
       ])
 
+  case .exercise:
+    return div(
+      timestampLinkView.view(block.timestamp)
```

Comment:
The one strange thing here is that we probably want them rendering one way in the body, another way at the end. Got any ideas?